### ISSUE TITLE
Use latest adapter 

### ISSUE THREAD

**Issue by denar90** [googlecodelabs/webrtc-web#2]
Hi. I noticed that `adapter` is in local folder. What about using latest [adapter](http://webrtc.github.io/adapter/adapter-latest.js)?
In `index.html` it will be like

```
....
<script src="//webrtc.github.io/adapter/adapter-latest.js"></script>
<script src="js/main.js"></script>
....
```

**Comment by aaronang** [comment]
I like this solution too.

@samdutton What are your thoughts on this? If you like this solution, I can submit a patch for this.

**Comment by samdutton** [comment]
@denar90 Thanks for picking up on this, and apologies for taking so long to respond. I think your suggestion is very sensible. I don't 

@aaronang I agree — and if you can do a patch, even better :).

**Comment by aaronang** [comment]
Working on it!

**Comment by samdutton** [comment]
👍👍

**Comment by nitobuendia** [comment]
@samdutton To be closed as per #61

**Comment by samdutton** [comment]
@nitobuendia thanks for the reminder. Closing.
